core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy

```diff
@@ -57,4 +57,14 @@ class BintrayConfiguration {
         }
     }
 
+    private void deriveDefaultsFromProject(Project project) {
+        if (extension.versionAttributes.isEmpty()) {
+            def gradlePluginPropertyFinder = new GradlePluginPropertyFinder(project)
+            String bestPluginId = gradlePluginPropertyFinder.findBestGradlePluginId()
+            if (bestPluginId != null) {
+                extension.versionAttributes << ['gradle-plugins': "$bestPluginId:$extension.groupId:$extension.artifactId"]
+                println "Using plugin identifier '" + extension.versionAttributes.get('gradle-plugins') + "' for gradle portal."
```

Comment:
They don't have the possibility to make a choice. We could add an extra
attributes if the user should have it.
Am 02.07.2015 10:59 schrieb "Paul Blundell" <notifications@github.com>:

> In
> core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy
> <https://github.com/novoda/bintray-release/pull/59#discussion_r33758624>:
>
> > @@ -57,4 +57,14 @@ class BintrayConfiguration {
> >          }
> >      }
> >
> > +    private void deriveDefaultsFromProject(Project project) {
> > +        if (extension.versionAttributes.isEmpty()) {
> > +            def gradlePluginPropertyFinder = new GradlePluginPropertyFinder(project)
> > +            String bestPluginId = gradlePluginPropertyFinder.findBestGradlePluginId()
> > +            if (bestPluginId != null) {
> > +                extension.versionAttributes << ['gradle-plugins': "$bestPluginId:$extension.groupId:$extension.artifactId"]
> > +                println "Using plugin identifier '" + extension.versionAttributes.get('gradle-plugins') + "' for gradle portal."
>
> I don't know. I wouldn't if they made the choice they know what they're
> doing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/bintray-release/pull/59/files#r33758624>.
>